app/src/test/java/org/groundplatform/android/usecases/survey/ListAvailableSurveysUseCaseTest.kt

```diff
@@ -49,19 +49,24 @@ class ListAvailableSurveysUseCaseTest : BaseHiltTest() {
     localSurveyStore.insertOrUpdateSurvey(survey)
   }
 
-  private fun setupSurveys(localSurveys: List<Survey>, remoteSurveys: List<Survey>) =
-    runWithTestDispatcher {
-      localSurveys.forEach { setupLocalSurvey(it) }
-      fakeRemoteDataStore.surveys = remoteSurveys
-    }
+  private fun setupSurveys(
+    localSurveys: List<Survey>,
+    remoteSurveys: List<Survey>,
+    publicSurveys: List<Survey>,
+  ) = runWithTestDispatcher {
+    localSurveys.forEach { setupLocalSurvey(it) }
+    fakeRemoteDataStore.surveys = remoteSurveys
+    fakeRemoteDataStore.publicSurveys = publicSurveys
```

Comment:
Why do we need a different field for public surveys? Instead, it should just be added to either `localSurveys` and/or `remoteSurveys` as needed.

Reply:
https://github.com/google/ground-android/pull/3107#discussion_r2063340464

Reply:
https://github.com/google/ground-android/pull/3107#discussion_r2063340464